src/rsem/rsem_prepare_reference/script.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+set -eo pipefail
+
+unset_if_false=( par_gff3_genes_as_transcripts par_polyA par_bowtie par_bowtie2 par_star par_hisat2_hca par_quiet par_prep_pRSEM )
+
+for par in ${unset_if_false[@]}; do
+    test_val="${!par}"
+    [[ "$test_val" == "false" ]] && unset $par
+done
+
+# replace ';' with ','
+par_reference_fasta_files=$(echo $par_reference_fasta_files | tr ';' ',')
+
+echo "$par_reference_fasta_files"
+rsem-prepare-reference \
+    ${par_gtf:+--gtf "${par_gtf}"} \
+    ${par_gff3:+--gff3 "${par_gff3}"} \
+    ${par_gff3_rna_patterns:+--gff3-RNA-patterns "${par_gff3_rna_patterns}"} \
+    ${par_gff3_genes_as_transcripts:+--gff3-genes-as-transcripts "${par_gff3_genes_as_transcripts}"} \
+    ${par_trusted_sources:+--gtrusted-sourcestf "${par_trusted_sources}"} \
```

Comment:
Fix refactoring error

```suggestion
    ${par_trusted_sources:+--trusted-sources "${par_trusted_sources}"} \